components/infobox/wikis/infobox_league_custom.lua

```diff
@@ -0,0 +1,121 @@
+---
+-- @Liquipedia
+-- wiki=pokemon
+-- page=Module:Infobox/League/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local League = require('Module:Infobox/League')
+local String = require('Module:StringUtils')
+local Variables = require('Module:Variables')
+local Class = require('Module:Class')
+local Injector = require('Module:Infobox/Widget/Injector')
+local Cell = require('Module:Infobox/Widget/Cell')
+local Title = require('Module:Infobox/Widget/Title')
+local Logic = require('Module:Logic')
+
+local CustomLeague = Class.new()
+local CustomInjector = Class.new(Injector)
+
+local _args
+local _game
+
+local _GAME = mw.loadData('Module:GameVersion')
+local _MODES = mw.loadData('Module:GameModes')
+local _FORMATS = mw.loadData('Module:GameFormats')
+
+function CustomLeague.run(frame)
+	local league = League(frame)
+	_args = league.args
+
+	league.addToLpdb = CustomLeague.addToLpdb
+	league.createWidgetInjector = CustomLeague.createWidgetInjector
+	league.defineCustomPageVariables = CustomLeague.defineCustomPageVariables
+	league.liquipediaTierHighlighted = CustomLeague.liquipediaTierHighlighted
+
+	return league:createInfobox(frame)
+end
+
+function CustomLeague:createWidgetInjector()
+	return CustomInjector()
+end
+
+function CustomInjector:parse(id, widgets)
+	if id == 'gamesettings' then
+		return {
+			Cell{name = 'Game version', content = {
+					CustomLeague._getGameVersion()
+				}
+			},
+			Cell{name = 'Game mode', content = {
+					CustomLeague:_getGameMode()
+				}
+			},
+			Cell{name = 'Format', content = {
+					CustomLeague:_getGameFormat()
+				}
+			},
+		}
+	elseif id == 'customcontent' then
+		if _args.player_number then
+			table.insert(widgets, Title{name = 'Players'})
+			table.insert(widgets, Cell{name = 'Number of players', content = {_args.player_number}})
+		end
+
+		--teams section
+		if _args.team_number then
```

Comment:
How should I correctly added the elseif (I tried the one in the pic [the purple highlighted text] but returns EOF expected) 
![image](https://user-images.githubusercontent.com/88981446/174623429-c3a874b3-6bba-4d2d-a0c3-c569b48c426c.png)

Reply:
kick the end above the elseif
the elseif continues from the if above, so the end of the above if breaks it

Reply:
noted, added 2nd revision. The check did return me error on **_game** after the elseif changes, not sure if its related

are the lines 22 24 90 (im not sure why because this werent being touched in previous revision and check were successful on that)

Reply:
fixed the issue for you ;)